nativeApp/src/volunteer_views/VolunteerRouter.tsx

```diff
@@ -0,0 +1,41 @@
+
+import React from 'react';
+import { createAppContainer } from 'react-navigation';
+import { createBottomTabNavigator } from 'react-navigation-tabs';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+
+import VolunteerHome from './screens/VolunteerHome';
+import Time from './screens/Time';
+import Settings from './screens/Settings';
+import { ColoursEnum } from '../lib/ui/colours';
+
+const TabNavigator = createBottomTabNavigator({
+  Home: VolunteerHome,
+  Time: Time,
+  Settings: Settings,
+},
+{
+  defaultNavigationOptions: ({ navigation }) => ({
+    tabBarIcon: ({tintColor }) => {
+      const { routeName } = navigation.state;
+      const IconComponent = MaterialCommunityIcons;
+      let iconName;
+      if (routeName === 'Home') {
+        iconName = "home-outline";
+      } else if (routeName === 'Time') {
+        iconName = `clock-fast`;
```

Comment:
Is eslint actually working? There are backticks here that should be `'`, the indentation is off, and the curly-bracket spacing on line 19 is inconsistent too.

Reply:
I think we may need to specifiy our own react native specific rules